### DISCUSSION TITLE
unable to `pip install jaxlib==0.3.14`

### DISCUSSION THREAD

**Discussion by davidcjuergens** [jax-ml/jax#20641] [Q&A] [answered]
Hello - I'm attempting to create a conda environment for [d3pm](https://github.com/google-research/google-research/tree/master/d3pm). I can install everything except `jaxlib==0.3.14` from requirements.txt.

I see on [pypi release history](https://pypi.org/project/jaxlib/#history) that 0.3.14 isn't in the history, which is seemingly the source of my installation error: 

```
ERROR: Could not find a version that satisfies the requirement jaxlib==0.3.14 (from versions: 0.4.6, 0.4.7, 0.4.9, 0.4.10, 0.4.11, 0.4.12, 0.4.13, 0.4.14, 0.4.16, 0.4.17, 0.4.18, 0.4.19, 0.4.20, 0.4.21, 0.4.22, 0.4.23, 0.4.24, 0.4.25, 0.4.26)
ERROR: No matching distribution found for jaxlib==0.3.14
```

Is there a way for me to get my hands on jaxlib==0.3.14? 

thanks for your help

**Comment by jakevdp** [comment, marked as answer]
Thanks for the question! Please see [FAQ: installing older jaxlib wheels](https://jax.readthedocs.io/en/latest/installation.html#installing-older-jaxlib-wheels) for advice on addressing your issue.

**Reply by davidcjuergens** [reply]
It worked perfectly, thank you Jake.
